Swift_Alert/Swift_Alert/CarsView.swift

```diff
@@ -0,0 +1,298 @@
+    private var carSeats = ["2 seats", "4 seats", "4 seats"]
+    private var carHoursePower = ["320 HP", "560 HP", "249 HP"]
+    private var carsImageName = ["audi", "bmw", "lexus"]
+    
+    // MARK: - Private Methods
+    
+    private var speedColor: some View {
+        Color.purple
+            .frame(
+                width: Constants.colorFrameWidth,
+                height: Constants.colorFrameHeight)
+            .cornerRadius(Constants.colorCornerRadius)
+            .offset(
+                x: -Constants.offsetX,
+                y: Constants.offsetY)
+    }
+    
+    private var speedImage: some View {
+        Image(Constants.speedImageName)
+            .resizable()
+            .frame(
+                width: Constants.imageFrame,
+                height: Constants.imageFrame)
+            .cornerRadius(Constants.colorCornerRadius)
+            .offset(
+                x: -Constants.offsetX,
+                y: Constants.imageOffsetY)
+    }
+    
+    private var speedText: some View {
+        Text("\(parameters[segmentIndex])")
+            .offset(
+                x: -Constants.offsetX,
+                y: Constants.textOffsetY)
+    }
+    
+    private var seatsColor: some View {
+        Color.purple
+            .frame(
+                width: Constants.colorFrameWidth,
+                height: Constants.colorFrameHeight)
+            .cornerRadius(Constants.colorCornerRadius)
+            .offset(
+                x: Constants.zeroForFrame,
+                y: Constants.offsetY)
+    }
+    
+    private var seatsImage: some View {
+        Image(Constants.seatsImageName)
+            .resizable()
+            .frame(
+                width: Constants.imageFrame,
+                height: Constants.imageFrame)
+            .cornerRadius(Constants.colorCornerRadius)
+            .offset(
+                x: Constants.zeroForFrame,
+                y: Constants.imageOffsetY)
+    }
+    
+    private var seatsText: some View {
+        Text("\(carSeats[segmentIndex])")
+            .offset(
+                x: Constants.zeroForFrame,
+                y: Constants.textOffsetY)
+    }
+    
+    private var pistonColor: some View {
+        Color.purple
+            .frame(
+                width: Constants.colorFrameWidth,
+                height: Constants.colorFrameHeight)
+            .cornerRadius(Constants.colorCornerRadius)
+            .offset(
+                x: Constants.offsetX,
+                y: Constants.offsetY)
+    }
+    
+    private var pistonImage: some View {
+        Image(Constants.pistonImageName)
+            .resizable()
+            .frame(
+                width: Constants.imageFrame,
+                height: Constants.imageFrame)
+            .cornerRadius(Constants.colorCornerRadius)
+            .offset(
+                x: Constants.offsetX,
+                y: Constants.imageOffsetY)
+    }
+    
+    private var pistonText: some View {
```

Comment:
```suggestion
    private var pistonTextView: some View {
```
   везде где тип some  View на конце View